paymentsheet/src/main/java/com/stripe/android/paymentsheet/wallet/SavedPaymentMethodsSheet.kt

```diff
@@ -0,0 +1,75 @@
+package com.stripe.android.paymentsheet.wallet
+
+import androidx.activity.ComponentActivity
+import androidx.annotation.RestrictTo
+import androidx.fragment.app.Fragment
+import com.stripe.android.ExperimentalSavedPaymentMethodsApi
+import com.stripe.android.paymentsheet.PaymentSheet
+import com.stripe.android.paymentsheet.repositories.CustomerAdapter
+
+/**
+ * 🏗 This features is under construction 🏗
+ *
+ * [SavedPaymentMethodsSheet] A class that presents a bottom sheet to manage a customer's
+ * saved payment methods.
+ */
+@ExperimentalSavedPaymentMethodsApi
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+interface SavedPaymentMethodsSheet {
+
+    /**
+     * Present's the saved payment methods sheet for the customer
+     */
+    fun present()
+
+    /**
+     * Configuration for [SavedPaymentMethodsSheet]
+     */
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+    class Configuration(
+        /**
+         * Your customer-facing business name.
+         *
+         * The default value is the name of your app.
+         */
+        val merchantDisplayName: String,
+
+        /**
+         * Describes the appearance of SavedPaymentMethodsSheet
```

Comment:
Nit pick... Sometimes we have comments as sentences, sometimes not. Sometimes we use `[]` to link to classes, and sometimes we don't. It would be great if we were consistent! I'd love to see [all comments as sentences](https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html), and all code references as links.